x-pack/plugins/security_solution/public/management/cypress/e2e/artifacts/artifacts_mocked_data.cy.ts

```diff
@@ -30,7 +30,7 @@ const loginWithoutAccess = (url: string) => {
   loadPage(url);
 };
 
-describe('Artifacts pages', { tags: '@ess' }, () => {
+describe('Artifacts pages', { tags: ['@ess', '@serverless', '@brokenInServerless'] }, () => {
```

Comment:
Is the test really broken in Serverless or you wanna just skip it for now? `@skipInServerless` tag could be a better choice, see [here](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/README.md#how-to-add-a-new-cypress-test)

Reply:
to me, it seems both tags are doing the same, maybe we could simplify and just have one tag for both use cases?